omnibus/Gemfile.lock

```diff
@@ -250,7 +243,6 @@ GEM
 
 PLATFORMS
   ruby
-  x86-mingw32
```

Comment:
`-  x86-mingw32`  The windows stuff seemed to get chopped off of the Gemfile.lock -- I am not sure why, but this seems like a mistake.